README.md

```diff
@@ -8,10 +8,37 @@
 
 This repo implements a pre-processing pipeline for the following documents. Currently, the pipeline is capable of recognizing the file type and choosing the relevant partition function to process the file.
 
-* Various plaintext files: `.txt`, `.eml`, `.html`, `.md`, `.json`
+* Plaintext: `.txt`, `.eml`, `.html`, `.md`, `.json`
 * Images: `.jpeg`, `.png`
 * Documents: `.doc`, `.docx`, `.ppt`, `.pptx`, `.pdf`
 
+## :rocket: Unstructured API
+
+Try our hosted API! It's freely available to use with any of the filetypes listed above. This is the easiest way to get started. If you'd like to host your own version of the API, jump down to the Developer Quickstart Guide.
```

Comment:
nit: can this anchor to `Developer Quickstart Guide`. admittedly it's not THAT far below, just a nicety?

Reply:
I opened a tab to look up how to do md anchors and then forgot to do it 😆

Reply:
🤣